docker/gaffer-jhub-options-server/.env

```diff
@@ -0,0 +1 @@
+JHUB_OPTIONS_SERVER_VERSION=latest
```

Comment:
The CI/CD scripts run `get-version.sh` to set `$JHUB_OPTIONS_SERVER_VERSION` before building the container. That script gets the version by parsing `package.json`. I don't really want us to have to remember to bump both `package.json` and `.env` on each new release. I would just delete the `.env` file, but then `docker-compose build` would break if users didn't run `get-version.sh` beforehand, so I decided to leave it there and just set the version to `latest` so that we don't have to keep updating it. What do you think? Leave it there as `latest`, leave it there and keep updating the version on each release, or just delete it and require users to run `get-version.sh` beforehand (or they could set the variable themselves)?

Reply:
oh, and yeah #138 will update the kubernetes helm chart so that the image tag used for the options server will match the version in package.json

Reply:
Can this be considered resolved then if #138 will look after this?